docs/release-notes.md

```diff
@@ -1,4 +1,18 @@
 ## 4.6.0
+##### Magpie Translation (MT)
+[Magpie Translation (MT)](https://github.com/zanata/zanata-mt/) provides the results from machine translation
+services.
+To enable MT, edits `standalone.xml` with following system properties:
+
+```xml
+<system-properties>
+  ...
+  <property name="mt.service.url" value="https://your.mt.service:8080"/>
```

Comment:
> https://your.mt.service:8080

That URL (https, port 8080) doesn't really make sense.

I would suggest https://mt.example.com/

Reply:
+1 on @seanf 's suggestion